gen/gen.go

```diff
@@ -176,10 +177,19 @@ fragment TypeRef on __Type {
   }
 }
 `
+	apiEndpoint := "https://" + apiHost + "/query"
+	endpoint, ok := os.LookupEnv("MONDOO_API_ENDPOINT")
+	if ok {
+		apiEndpoint, err = url.JoinPath(endpoint, "/query")
```

Comment:
we should apply that logic to the default api host too.

Reply:
Fixed.